### ISSUE TITLE
Bit Emote Support

### ISSUE THREAD

**Issue by swemoney** [chatty/chatty#372]
**Is your feature request related to a problem? Please describe.**
Not all emotes for a channel appear in the emote window. Emotes that are unlocked by hitting bit milestones are not available to select in the emote window. Typing out the emote name inside chatty also doesn't show the emote when the chat is sent but if someone sends that emote from twitch chat, the emote appears in chat just fine.

**Describe the solution you'd like**
The ability to select bit milestone emotes from the emote windows as well as the ability to auto-complete them using :emotename.

**Additional context**
![bit emote context](https://user-images.githubusercontent.com/4160/82133730-124afb00-97b5-11ea-8354-1b4673b42b49.png)
In this image, the first emote was sent by Twitch chat. Then I typed the same emote into Chatty.

**Comment by tduva** [comment]
Bit Emotes are currently not supported by the Twitch API.

**Comment by dungi** [comment]
yes, would like to see that too. If I write the Emote currently, it's just the text.
